### ISSUE TITLE
ffmpeg with VAAPI will not work without installing mesa-va-drivers

### ISSUE THREAD

**Issue by richardbrodie** [HaveAGitGat/Tdarr#397]
**Describe the bug**
Using any VAAPI accelerated ffmpeg command in the docker container, e.g. when transcoding, fails. Installing mesa-va-drivers solves the problem.

**To Reproduce**
1) run `docker-compose exec <tdarr_node_container> bash`
2) run `ffmpeg -init_hw_device vaapi=foo:/dev/dri/renderD128`
3) receive output: 
```
[AVHWDeviceContext @ 0x55b7541a9180] Failed to initialise VAAPI connection: -1 (unknown libva error).
Device creation failed: -5.
Failed to set value 'vaapi=foo:/dev/dri/renderD128' for option 'init_hw_device': Input/output error
Error parsing global options: Input/output error
```
4) run `apt install mesa-va-drivers`
5) run `ffmpeg -init_hw_device vaapi=foo:/dev/dri/renderD128 ... `
3) ffmpeg succeeds

**Expected behavior**
ffmpeg command succeeds by default

 - OS: linux
 - Browser chrome
 - Version 2.00.06

**Comment by HaveAGitGat** [comment]
Hi thanks for this, I've been wanting to add this in for a while but I don't have a vaapi capable machine which makes things difficult.

I've added the change to the dev containers  (`tdarr:dev` and `tdarr_node:dev`)

Would you mind verifying it's working?
Thanks

**Comment by richardbrodie** [comment]
Seems to work now. Thanks for getting to this so quickly.

**Comment by HaveAGitGat** [comment]
No problem, thanks for posting the fix! It's in the latest containers now (2.00.07).
